Unable to import local files

# This is a Bug Report
## Description

When I import my local files I have to specify the file extension otherwise webpack throws an error

`import { Student } from "./student.ts";`
Works fine 👍 

`import { Student } from "./student";`
Throws error:
`ERROR in ./handler.ts
Module not found: Error: Can't resolve './student' in '/Users/edwardbyrne/Desktop/my-service'
 @ ./handler.ts 3:16-36
`

 Very strange!

## Additional Data
```

webpack.config.js
const path = require('path');
// eslint-disable-next-line import/no-unresolved
const slsw = require('serverless-webpack');

module.exports = {
  entry: slsw.lib.entries,
  output: {
    libraryTarget: 'commonjs',
    path: path.join(__dirname, '.webpack'),
    filename: '[name].js',
  },
  target: 'node',
  module: {
    loaders: [
      { test: /\.ts(x?)$/, loader: 'ts-loader' },
    ],
  },
};

```

--- a/examples/typescript/package.json
+++ b/examples/typescript/package.json
@@ -10,8 +10,8 @@
   "license": "MIT",
   "devDependencies": {
     "serverless-webpack": "file:../../",
-    "ts-loader": "^2.3.1",
-    "typescript": "^2.4.2",
-    "webpack": "^3.3.0"
+    "ts-loader": "^2.3.7",
+    "typescript": "^2.5.2",
+    "webpack": "^3.6.0"
   }
 }

--- a/examples/typescript/webpack.config.js
+++ b/examples/typescript/webpack.config.js
@@ -5,9 +5,6 @@ module.exports = {
   entry: slsw.lib.entries,
   resolve: {
     extensions: [
-      '',
-      '.webpack.js',
-      '.web.js',
       '.js',
       '.json',
       '.ts',

--- a/examples/typescript/webpack.config.js
+++ b/examples/typescript/webpack.config.js
@@ -3,6 +3,9 @@ const slsw = require('serverless-webpack');
 
 module.exports = {
   entry: slsw.lib.entries,
+  resolve: {
+    extensions: ['.ts', '.js', '.tsx', '.jsx', '.json'],
+  },
   output: {
     libraryTarget: 'commonjs',
     path: path.join(__dirname, '.webpack'),

--- a/examples/typescript/webpack.config.js
+++ b/examples/typescript/webpack.config.js
@@ -1,10 +1,18 @@
-var path = require('path');
+const path = require('path');
 const slsw = require('serverless-webpack');
 
 module.exports = {
   entry: slsw.lib.entries,
   resolve: {
-    extensions: ['.ts', '.js', '.tsx', '.jsx', '.json'],
+    extensions: [
+      '',
+      '.webpack.js',
+      '.web.js',
+      '.js',
+      '.json',
+      '.ts',
+      '.tsx'
+    ]
   },
   output: {
     libraryTarget: 'commonjs',

